load-tests/browse_homepage.js

```diff
@@ -0,0 +1,133 @@
+import { check, sleep, group } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+    userAgent: 'k6LoadTest',
+};
+
+const check2xx = res => {
+    const resArray = Array.isArray(res) ? res : [res];
+    resArray.forEach(res =>
+        check(res, {
+            'is status 2xx': (r) => r.status >= 200 && r.status < 300,
+        })
+    );
+};
+
+const referer = 'https://staging.openapparel.org/';
+
+// rootUrl should NOT have a trailing slash
+const rootUrl = 'https://staging.openapparel.org';
+
+const staticHeaders = {
+    'sec-ch-ua': '" Not A;Brand";v="99", "Chromium";v="99", "Google Chrome";v="99"',
+    'sec-ch-ua-mobile': '?0',
+    'sec-ch-ua-platform': '"macOS"',
+    referer,
+};
+
+const apiHeaders = {
+    accept: 'application/json, text/plain, */*',
+    'x-oar-client-key': __ENV.CLIENT_KEY || 'NOT_SPECIFIED',
+    credentials: 'same-origin',
+    'sec-ch-ua': '" Not A;Brand";v="99", "Chromium";v="99", "Google Chrome";v="99"',
+    'sec-ch-ua-mobile': '?0',
+    'sec-ch-ua-platform': '"macOS"',
+    referer,
+};
+
+const buildGet = (url, headers) => {
+    return {
+        method: 'GET',
+        url,
+        params: {
+            headers: headers || staticHeaders,
+        }
+    };
+};
+
+export default function main() {
+    let responses;
+
+    // Wait up to 5 seconds so that the iterations aren't exactly overlapping
+    sleep(Math.random() * 5);
+
+    group('https://staging.openapparel.org/', function () {
+        responses = http.batch([
+            buildGet(`${rootUrl}/web/environment.js`),
+            buildGet(`${rootUrl}/static/css/2.037bc208.chunk.css`),
+            buildGet(`${rootUrl}/static/css/main.cfc7f5ef.chunk.css`),
+            buildGet(`${rootUrl}/static/js/2.b90b8892.chunk.js`),
+            buildGet(`${rootUrl}/static/js/main.058b1d83.chunk.js`),
```

Comment:
These are going to be different for each release. I think the way to get these dynamically is to search the HTML returned from a request to `/`